ex_cubic_ods_ingestion/lib/ex_cubic_ods_ingestion/repo.ex

```diff
@@ -24,8 +24,17 @@ defmodule ExCubicOdsIngestion.Repo do
       port = Keyword.fetch!(config, :port)
       token = aws_rds.generate_db_auth_token(hostname, username, port, %{})
 
-      # update password with token
-      Keyword.put(config, :password, token)
+      # update password with token and update ssl options (if set) to ref rds cert
+      Keyword.merge(config,
+        password: token,
+        ssl_opts: [
+          cacertfile: "priv/aws-cert-bundle.pem",
```

Comment:
suggestion: when running as a release, I'm not sure that "priv" is relative to the working directory in the same way.

```suggestion
          cacertfile: Application.app_dir(:ex_cubic_ods_ingestion, ["priv", "aws-cert-bundle.pem"]),
```

Reply:
I deployed to dev and it seems to work, but I agree that your suggestion is more fool proof and have applied it. Thanks.